THIRD_PARTY_LICENSES.txt

```diff
@@ -5324,7 +5324,7 @@ Copyright 2014 The Go Authors.  All rights reserved.
 Copyright 2014 The Go Authors. All rights reserved.
 Copyright 2014 The Kubernetes Authors.
 Copyright 2014 The go-github AUTHORS. All rights reserved.
-Copyright 2014 Unknwon
+Copyright 2014 Unknown
```

Comment:
Lawyers tend to get annoyed at changes to license files...

That said, unless there's an entity named `Unknwon`, this is problematic

Reply:
Apparently this is https://github.com/Unknwon (presumably Grafana or similar). It's unfortunate that it isn't an email address or name as opposed to an amorphous thing, but whatever. Dropping.